integr8ly/integreatly-operator
# scale down fuse deployments

# Issue link
Scale down fuse pods and invert alerts such that they will fire if fuse pods are running. 

# Verification steps
Install 2.9.3
Verify Fuse pods running 
Install from this branch, verify that fuse pods are scaled down and no alerts are firing.

Stop the RHMI operator. 
Scale up the Fuse pods
Verify after a period alerts are firing

Start the operator again
Verify pods scale down to zero again
Verify no alerts firing
